### PR TITLE
Move inline editing methods into a separate service

### PR DESCRIPTION
**Pull request by thoessl**
This moves about a hundred lines from the MotionDetailCtrl into a separate service in a new file.
Extra testing if everything still works as intended would be appreciated :-)

The PDF-function would be another candidate for a separate service in this file.

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@thoessl Please rebase and resolve the conflicts. Then I can test your pr with current master.

File: openslides/motions/static/js/motions/motion-services.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+(function () {
+
+"use strict";
+
+angular.module('OpenSlidesApp.motions.motionservices', ['OpenSlidesApp.motions', 'OpenSlidesApp.motions.lineNumbering'])
+
+    .service('MotionInlineEditing', [
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/motion-services.js]: This indention is not necessary, I think.

Comment (normanjaeckel) [inline openslides/motions/static/js/motions/motion-services.js]: Is there a reason why you use a service and not a factory? The difference is, I think, that a service uses the returned object as constructor and uses `new` whereas a factory just returns the object. We do not need the `new` construction here, do we?

**Comment by thoessl** [comment]
Ok, I made a factory out of the service and reduced the indentation.

**Comment by emanuelschuetze** [comment]
Ok, inline editing successfully tested. Ready for merge.

**Pull request merged**